modules/database/sqlite/db.copy

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Can we rename this file to give it a `.php` extension? I know it shouldn't be used directly due to the placeholders, but it would be nice to get the regular syntax highlighting etc. I don't see a good reason for keeping off the proper extension. We could name it `db.copy.php`.

Looking at the file, I'm pretty sure there is technically no invalid PHP in here, the only placeholder is in the path string, so that should be fine from a PHP linting perspective.

Reply:
Done